### PR TITLE
[260] add test cases for HabitStatisticController

### PR DESCRIPTION
**Pull request by IliaWithHat**

### CODE REVIEW

**Review by tellmant — APPROVED** [review]

**Review by tellmant — APPROVED** [review]

**Review by RockTF — APPROVED** [review]

File: core/src/test/java/greencity/controller/HabitStatisticControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    @BeforeEach
+    void setUp() {
+        this.mockMvc = MockMvcBuilders
+                .standaloneSetup(habitStatisticController)
+                .setCustomArgumentResolvers(new UserArgumentResolver(userService, modelMapper))
+                .build();
+    }
+
+    @Test
+    void findAllByHabitIdTest() throws Exception {
```
Comment (nataliiapovzhyk) [inline core/src/test/java/greencity/controller/HabitStatisticControllerTest.java:62]: Annotation Display Name is missing

File: core/src/test/java/greencity/controller/HabitStatisticControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        mockMvc.perform(get(habitStatisticLink + "/{habitId}", 1))
+                .andExpect(status().isOk());
+        verify(habitStatisticService).findAllStatsByHabitId(1L);
+    }
+
+    @Test
+    void findAllStatsByHabitAssignIdTest() throws Exception {
```
Comment (nataliiapovzhyk) [inline core/src/test/java/greencity/controller/HabitStatisticControllerTest.java:69]: Annotation Display Name is missing

**Review by nataliiapovzhyk — COMMENTED** [review]
Missing Display Name for tests

**Review by df7734 — APPROVED** [review]

**Review by nataliiapovzhyk — APPROVED** [review]
test passes

**Pull request merged**